docs/java/java-testing.md

```diff
@@ -13,9 +13,9 @@ MetaSocialImage:
 
 Testing Java in Visual Studio Code is enabled by the [Java Test Runner](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test) extension. It's a lightweight extension to run and debug Java test cases. The extension supports the following test frameworks:
 
-- JUnit 4 (v4.8.0+)
-- JUnit 5 (v5.1.0+)
-- TestNG (v6.8.0+)
+- [JUnit 4](https://junit.org/junit4/) (v4.8.0+)
+- [JUnit 5](https://junit.org/junit5/) (v5.1.0+)
+- [TestNG](https://testng.org/doc/download.html) (v6.8.0+)
```

Comment:
Hi @hexiaokai. Is there any reason only the TestNG is directed to the download page?

Reply:
good catch. updated the link.